allure-report-face/src/components/pane-set/styles.css

```diff
@@ -64,3 +63,7 @@
     }
   }
 }
+
+.pane + .pane {
+  box-shadow: 0 0 3px #eee;
+}
```

Comment:
okay

But it is still worse that it was before. Shadows are not good here :weary: